lab-isaac/lib/error-middleware.js

```diff
@@ -0,0 +1,18 @@
+'use strict';
```

Comment:
You may have to refactor this module a bit so it can catch the mongo errors as well.